skins/laika/src/components/pages/donation_confirmation/MembershipInfo.vue

```diff
@@ -3,9 +3,21 @@
         <div class="column is-half">
             <h2 class="title is-size-5">{{ $t( 'donation_confirmation_membership_call_to_action_title' ) }}</h2>
             <p>{{ $t( 'donation_confirmation_membership_call_to_action_text' ) }}</p>
-            <p><b-button type="is-primary is-main">{{ $t('donation_confirmation_membership_button') }}</b-button></p>
+            <p>
+                <a :href="membershipApplicationUrl">
+                    <b-button type="is-primary is-main has-margin-top-36">{{ $t('donation_confirmation_membership_button') }}</b-button>
+                </a>
+            </p>
+        </div>
+        <div class="column is-half">
+            <ul>
+                <li>{{ $t( 'donation_confirmation_bottombox_membership_benefit_1' ) }}</li>
```

Comment:
The list bullets are still in the messages text and not in the CSS for the list items. Having `<li>` elements is already semantically better, but not from a typography/aestetics stand point. Would you mind making that change?